### PR TITLE
fix(fastify-template): fix for subscriptions in fastify template

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Fix for #2174 

## Types of changes

What types of changes does your code introduce to Graphback?

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please specify)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/aerogear/graphback/blob/master/CONTRIBUTING.md) document.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

As [discussed](https://github.com/aerogear/graphback/issues/2174#issuecomment-752543660) I've added resolvers in the plugin options of mercurius which should fix the issue.

## Verification

I tried verifying this by enabling the fastify template and then using the updated cli (by running `node dist/index.js` in the create-graphback folder) but I received this error due to which the template didn't load and I wasn't able to test it. Please let me know if there are some other steps I can take to test if the subscriptions are working or not. Thanks.
sendNew

### CODE REVIEW

File: templates/ts-fastify-mongodb-backend/src/index.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ async function start() {
 
   app.register((mercurius as any), {
     schema,
+    resolvers,
```
Comment (machi1990) [inline templates/ts-fastify-mongodb-backend/src/index.ts:48]: I think `resolvers` should include `noteResolvers` as in https://github.com/aerogear/graphback/blob/12d588718dacaecb5cc1d3ed8005ac4a949b82a4/templates/ts-fastify-mongodb-backend/src/index.ts#L40

**Review by machi1990 — CHANGES_REQUESTED** [review]
Thanks for the PR. I've not verified it as I am away from laptop but it LGTM barring the small requested change inline. 

I'll let @craicoverflow @wtrocki do the verification.

**Comment by wtrocki** [comment]
Let's start this year with this small PR. Looking at it now

**Comment by wtrocki** [comment]
```
 ! [remote rejected]       pr/RinkiyaKeDad/2248-1 -> pr/RinkiyaKeDad/2248-1 (permission denied)
error: failed to push some refs to 'git@github.com:RinkiyaKeDad/graphback'
```

I cannot fix issues in PR. Can you add me to fork :) THX

**Comment by wtrocki** [comment]
> I tried verifying this by enabling the fastify template and then using the updated cli (by running

How about running this from the code:
```
cd templates/ts-fastify-mongodb-backend
docker-compose up -d
yarn develop
```

PS: This doesn't seem to be working for me.

**Comment by RinkiyaKeDad** [comment]
> I think `resolvers` should include `noteResolvers` as in

Thanks for pointing that out 😅 I've added them now @machi1990.




> Can you add me to fork :) THX

Added you @wtrocki :)







> PS: This doesn't seem to be working for me.




Same. It doesn't give an error in the terminal like last time, but shows that localhost refused to connect on the browser.



NewTest

**Comment by RinkiyaKeDad** [comment]
> How about running this from the code:

Okay so this way of testing the template works if I don't specify the noteResolvers and simply leave the code like this:
```
  app.register((mercurius as any), {
    schema,
    resolvers,
    graphiql: true,
    context: contextCreator,
    subscription: true
  });
```
But even with this when I tested the subscriptions they were not working. Could it be because [`makeExecutableSchema`](https://github.com/aerogear/graphback/blob/master/templates/ts-fastify-mongodb-backend/src/index.ts#L39) is not creating the schema in a way acceptable by mercurius?

Will have to look for some other way to fix this then I guess 🤔

**Comment by RinkiyaKeDad** [comment]
Would it be a good idea to switch from mercurius to [apollo-server-fastify](https://github.com/apollographql/apollo-server/tree/main/packages/apollo-server-fastify)?

**Comment by wtrocki** [comment]
Sounds good?

**Comment by RinkiyaKeDad** [comment]
Closing this since we're shifting to apollo-server-fastify. See #2256

**Pull request closed without merging**